### PR TITLE
Check if current tmux pane is active

### PR DESCRIPTION
**Pull request by grimpy**
Signed-off-by: Jo De Boeck <deboeck.jo@gmail.com>

### CODE REVIEW

File: conf.d/done.fish [hunk 1]
```diff
@@ -31,6 +31,20 @@ function __done_get_window_id
 	end
 end
 
+function __done_tmux_active
+	if type -q tmux
+		set tmuxpanes (tmux list-panes -a -F '#{pane_pid}')
+		set mypid %self
+		for tmuxpid in $tmuxpanes
+			if test $tmuxpid -eq $mypid
+				tmux list-panes -F '#{pane_active} #{pane_pid}' | grep "^1 $mypid" > /dev/null
```
Comment (franciscolourenco) [inline conf.d/done.fish]: Why iterate over the available panes? Wouldn't it be enough to return 
`tmux list-panes -F '#{pane_active} #{pane_pid}' | grep "^1 $mypid" > /dev/null` status?

**Comment by grimpy** [comment]
Rebased + reworked based on the previous comment

**Comment by grimpy** [comment]
Only when the window is not active, I'm assuming that one can always see
all panes in the active window...

**Comment by franciscolourenco** [comment]
@grimpy thanks for the contribution <3 Did a bit of refactoring, could you please install done 1.7 and let me know if something is not working as expected? thank you!

**Comment by ammgws** [comment]
@grimpy Are you still using this plugin?

**Comment by grimpy** [comment]
@ammgws I'm not even using fish atm

**Comment by ammgws** [comment]
@grimpy You might remember that this script only sends notifications if the tmux window is inactive. In my case I also want it to send notifications when the session is detached, and am thinking about sending a PR for that. If you were still using fish with this plugin, would that have been a problem for your workflow?

**Comment by grimpy** [comment]
It wouldn't and it sounds like a cool idea

**Pull request merged**